### ISSUE TITLE
Add JS linter

### ISSUE THREAD

**Issue by mmmavis** [Pomax/react-formbuilder#47]
(placeholder ticket, will add more details here soon)

**Comment by mmmavis** [comment]
@Pomax How do we wanna use Prettier? Do we want it to lint and fix things in place? I was thinking to have a precommit script that runs something like `prettier --write [other options]`. Thoughts?

**Comment by Pomax** [comment]
yeah: force it. Run the linter with in-place rewriting before a commit and that's a worry no one has to have anymore.
